src/utils.ts

```diff
@@ -76,8 +76,9 @@ export function error(...messages: any[]) {
   console.error(whiteBright.bgRedBright('error'), ...messages)
 }
 
+const debugEnabled = process.env.DEBUG !== undefined && process.env.DEBUG.includes('json2ts')
```

Comment:
I used `.includes` here instead of a direct equality check in order to make this compatible with the `debug` module, since the `DEBUG` env var as used in that package encourages users to separate namespaces or patterns using a comma or space, so in my case I might do something like the following:

env DEBUG=react,request,json2ts